R/utility.R

```diff
@@ -326,24 +328,19 @@ waitForNodesToComplete <- function(poolId, timeout = 86400) {
 #' getJobResult(jobId = "job-001")
 #' }
 #' @export
-getJobResult <- function(jobId = "", ...) {
-  args <- list(...)
+getJobResult <- function(jobId = "") {
+  cat("Getting job results...", fill = TRUE)
 
-  print("Getting job results...")
+  tempFile <- tempFile <- tempfile("getJobResult", fileext = ".rds")
 
-  if (!is.null(args$container)) {
-    results <-
-      rAzureBatch::downloadBlob(args$container,
-                                paste0("result/", jobId, "-merge-result.rds"))
-  }
-  else{
-    results <-
-      rAzureBatch::downloadBlob(jobId, paste0("result/", jobId, "-merge-result.rds"))
-  }
+  results <- rAzureBatch::downloadBlob(
```

Comment:
Should we catch the result if the result is not found?

Reply:
Actually if the result is not found, we don't get an exception, instead the error message is returned as results, that's why I have a is.vector(results) later to detect if an error is encountered.